netcost.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM golang:latest
+LABEL maintainer="Alban Crequy <alban@kinvolk.io>"
+
+ADD netcost /bin/
+CMD ["/bin/netcost", "-ifindex", "3", "-netlist", "192.168.0.0/16,0.0.0.0/0"]
```

Comment:
Any reason to have those default values different than the ones in the netcost executable?